calculators/referendum_supporters_calculator.rb

```diff
@@ -77,4 +118,29 @@ def committee_from_expenditure(expenditure)
 
     committee
   end
+
+  # Form 497 Page 2 (Late Expenditures) includes the ballot measure name and
+  # committee ID, but does not indicate whether that expenditure was in support
+  # or opposition of the ballot measure.
+  #
+  # This is not perfect, but it should get us pretty close.
+  def guess_whether_committee_supports_measure(committee_id, bal_name)
+    @_guess_cache ||=
+      begin
+        guesses = ActiveRecord::Base.connection.execute(<<-SQL)
+          SELECT "Filer_ID", "Bal_Name", "Sup_Opp_Cd"
```

Comment:
Yeah, I didn't use the Report_Num here because it seemed unlikely to me that the Sup_Opp_Cd would change between amendments.